src/Views/Nav.elm

```diff
@@ -29,12 +29,6 @@ view model =
             , navItem (t IconContact) (t ContactLinkDestination) "contact" "email" (t ContactLinkShort) (t ContactLinkLong)
             , span [ class "nav-item mobile-only" ]
                 [ callToActionNavItemMobile (callToActionConstructor Survey (t CallToActionDestination) (t CallToActionDestinationDisplay)) ]
-            , span [ class "nav-item" ]
```

Comment:
If you run elm-analyse now does it get rid of some unused imports?

Reply:
There are some unused imported variables but none which seem to be as a result of my meddling? Comparing the main branch to this one there are no new unused variables, although that could be just because I'm not familiar with elm-analyse as you guys and I might be missing a trick or ten...

Reply:
:+1: OK we'll run another clean at some point.